src/SubscriptionModule.sol

```diff
@@ -65,6 +65,8 @@ contract SubscriptionModule {
 
     event RecipientUpdated(bytes32 indexed id, address indexed oldRecipient, address indexed newRecipient);
 
+    event Unsubscribed(bytes32 indexed id, address indexed subscriber);
```

Comment:
Not sure if we need the subscriber. Is there anything you plan on using it for? It's also totally fine to keep. Just wondering 🤔